web/crux-ui/.env.example

```diff
@@ -2,7 +2,7 @@ CRUX_API_ADDRESS=172.17.0.1:5001
 CRUX_INSECURE=true
 
 KRATOS_URL=http://172.17.0.1:8000/kratos
-KRATOS_ADMIN_URL=http://172.17.0.1:9434
+KRATOS_ADMIN_URL=http://172.17.0.1:4434
```

Comment:
Change these to localhost as well please!

Reply:
that's gonna mess up cases where crux is running in a container

Reply:
localhost is meh on linux, 172.x.x.x  won't fly with windows.

Reply:
I will add a comment then.

Reply:
@m8vago but you shouldn't connect  to a container directly, if you have some business with those, expose them properly, then it will work on all platforms. Even then, 172.17.0.1 is only feasible if your dev machine have the default docker IP subnet. If you have some magic in your net stack, you can't use it.